packages/centreon-ui/src/utils/useCopyToClipboard.ts

```diff
@@ -0,0 +1,33 @@
+import useSnackbar from '../Snackbar/useSnackbar';
+
+type CopyFn = (text: string) => void;
```

Comment:
```suggestion
type CopyFunction = (text: string) => void;
```

Prefer explicit names over shortcuts :)